src/BenchmarkDotNet/Templates/BuildPlots.R

```diff
@@ -10,6 +10,17 @@ library(tidyr)
 library(grid)
 library(gridExtra)
 
+isEmpty <- function(val){
+   is.null(val) || val == ""
+}
+
+verifyParams <- function(params){ 
+  value = "Default"
```

Comment:
Please, use `<-` instead of `=` for assignment operations.

Reply:
@AndreyAkinshin, please look my last changes. Thank you for reviewing.

Reply:
You still use `=` as the assignment operator. Please, use `<-` instead.

Reply:
it's weird, I replaced assignments operator just for my code: https://github.com/dotnet/BenchmarkDotNet/pull/1212/commits/82c8abb60acc4b711f114e6ee995da28ba41f20f. Should I replaced all others `<-` in R.script?